Sources/SwiftKotlinFramework/Utilities.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Utilities.swift
+//  SwiftKotlinFramework
+//
+//  Created by Tor Langballe 07/07/2018.
+//
+
+import AST
+import Transform
+
+func firstCharIsUpper(str:String) -> Bool {
+    let s = String(str.first!)
+    return s == s.uppercased()
```

Comment:
What if we rewrite it as an extension on a char that cheks if it is upper case? then you could do `string[0].isUpperCase` or `string.first?.isUpperCase`